options/locale/locale_en-US.ini

```diff
@@ -2815,6 +2815,7 @@ monitor.process = Running Processes
 monitor.desc = Description
 monitor.start = Start Time
 monitor.execute_time = Execution Time
+monitor.last_execution_result = Success?
```

Comment:
The text could be `Result`. `Success?` reads a little strange on UI in my mind.

Reply:
```suggestion
monitor.last_execution_result = Result
```